op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -156,9 +157,13 @@ func Test_Operator_Fee_Constistency(gt *testing.T) {
 
 			aliceInitialBalance, l1FeeVaultInitialBalance, baseFeeVaultInitialBalance, sequencerFeeVaultInitialBalance, operatorFeeVaultInitialBalance = getCurrentBalances()
 
+			bobInitialBalance := balanceAt(env.Bob.Address())
+
 			// regular Deposit, in new L1 block
 			env.Alice.L1.ActResetTxOpts(t)
-			env.Alice.L2.ActSetTxValue(new(big.Int).SetUint64(100000))
+			env.Alice.L2.ActResetTxOpts(t)
+			env.Alice.L2.ActSetTxToAddr(&env.Dp.Addresses.Bob)(t)
```

Comment:
Curious how much of this we actually need given we set the "to" to Bob above already. I don't think resetting the `TxOpts` resets the "to", and I am also not sure if we need to reset the `TxOpts`. So we could potentially removed L163-L165 (inc)?